app/views/common/_form_autocomplete.html.erb

```diff
@@ -0,0 +1,9 @@
+<div id="<%= options[:control_group_id] %>"></div>
+  <%=mount_react_component('FormAutocomplete', "##{options[:control_group_id]}", {
```

Comment:
with this architecture, only one autocomplete field actually works in a page (adding more fields ends up with duplicating data from the first field), IMHO we should support in multiple autocomplete fields.

Reply:
fixed :)